x-pack/plugins/observability_solution/apm/server/routes/dependencies/get_top_dependency_spans.ts

```diff
@@ -119,9 +117,10 @@ export async function getTopDependencySpans({
     })
   ).hits.hits.map((hit) => unflattenKnownApmEventFields(hit.fields, topDedsRequiredFields));
 
-  const transactionIds = spans.map((span) => span.transaction.id);
+  const traceIds = spans.map((span) => span.trace.id);
```

Comment:
What's the impact of changing it to look at `trace` instead?

Reply:
As it's explain in the description, span documents in otel don't have transaction.id, so we search for trace id instead

Reply:
Could that affect Elastic APM data?

Reply:
We might have spans that are not transactions 🤔